backend_simple.py

```
@@ -304,6 +345,63 @@ async def get_me(current_user: Dict = Depends(get_current_user)):
         created_at=datetime.fromisoformat(current_user["created_at"])
     )
 
+@app.get("/api/v1/trading/portfolio", response_model=PortfolioResponse)
+async def get_portfolio(current_user: Dict = Depends(get_current_user)):
```

Ruff (0.12.2)

Unused function argument: `current_user`

(ARG001)